src/community.rs

```diff
@@ -0,0 +1,54 @@
+use near_sdk::borsh::{self, BorshDeserialize, BorshSerialize};
+use near_sdk::serde::{Deserialize, Serialize};
+use near_sdk::{env, AccountId};
+
+#[derive(BorshSerialize, BorshDeserialize, Serialize, Deserialize, Clone)]
+#[serde(crate = "near_sdk::serde")]
+pub struct Github {
+    pub repo: String,
+    pub labels: Vec<String>,
+}
+
+#[derive(BorshSerialize, BorshDeserialize, Serialize, Deserialize, Clone)]
+#[serde(crate = "near_sdk::serde")]
+pub struct Community {
+    pub name: String,
+    pub description: String,
+    pub image: String,
+    pub thumbnail: String,
+    pub overview: String,
+    pub events: String,
+    pub admins: Vec<AccountId>,
+    pub labels: Vec<String>,
+    pub telegram: Vec<String>,
```

Comment:
I would use `community_group_urls` and have a vector of URLs instead of hardcoding telegram, as some communities may choose Discord, Reddit, WeChat, etc.

Reply:
Telegram is a special one required in the user story - to render a UI of telegram messages. @ori-near Do we need other form of community group urls?

Reply:
So it is a telegram chat ID or telegram handle, and not URL? Please, clarify that with the field name

Reply:
Should be the telegram channel link:

<img width="450" alt="image" src="https://user-images.githubusercontent.com/13259400/235149465-8d83d901-c2e7-4259-87b7-fcfe9fe207f5.png">

Reply:
@ailisp Please, consider renaming the field to `telegram_handle` to avoid confusion